docs/getting_started/your_first_application.md

```diff
@@ -55,4 +55,7 @@ int main()
     app.port(18080).multithreaded().run();
 }
 ```
-After building and running your .cpp file, you should be able to access your endpoint at [http://localhost:18080](http://localhost:18080). Opening this URL in your browser will show a white screen with "Hello world" typed on it.
+
+For the sake of simplicity, we suggest using `crow_all.h` (by placing it in the same place as your `main.cpp`) and running the command `g++ main.cpp -lpthread -O2 -o first` (use `clang` instead of `g++` if you prefer clang, or Visual Studio if you're on windows).
+
+After building and running your `.cpp` file, you should be able to access your endpoint at [http://localhost:18080](http://localhost:18080). Opening this URL in your browser will show a white screen with "Hello world" typed on it.
```

Comment:
`your app file` (`first`)? Instead of `.cpp` - if we assume that someone need basic command to compile, then we should also precisely point to exec ;)

Reply:
I'm a little confused, what are you suggesting instead of `first`?

Reply:
`After building and running your `.cpp` file, you should be able to access your endpoint` -> `After building and running your app (first) file, you should be able to access your endpoint`